roles.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "type": "object",
+  "properties": {
+    "roleName": {
+      "type": "string"
+    },
+    "displayName": {
+      "type": "string"
+    },
+    "verbs": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "serviceVerbs": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "isInternal": {
+      "type": "boolean"
+    },
+    "isDeleted": {
+      "type": "boolean"
+    },
+    "allowedDomains": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "allowedLocationLabels": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    }
+  },
+  "required": [
```

Comment:
Are all of them really required? I guess we want to have that for now unless we want to handle `null` serviceVerbs?

Reply:
I'm not sure, thought that there were required but maybe not all of them?

Reply:
Maybe they are for now so let's stick with this. We did have some issues were people put `null` as verbs right?

Reply:
Yea, although I think we now have fixed the null issues that we had previously with null checks, it would be nice not to encounter any similar issues in the future. If we can regulate it here it to me seems like a good idea

Reply:
Then we should enforce to make it required here?

Reply:
Yeah, let's start with that.